imagecraft/pack/chroot.py

```diff
@@ -107,6 +104,7 @@ def _runner(
     logger.debug("[pid=%d] child process: target=%r", pid, target)
     try:
         logger.debug("[pid=%d] chroot to %r", pid, path)
+        os.chdir(path)
```

Comment:
Why are we changing to the directory before the chroot? Maybe we should do it after the chroot (changing to `/`) to ensure cwd is significant after the change?